src/main/java/org/mamute/dao/QuestionDAO.java

```diff
@@ -278,5 +277,126 @@ public Question questionWith(Attachment attachment) {
 				.setParameter("a", attachment)
 				.uniqueResult();
 	}
-}
 
+	public void delete(Question question) {
```

Comment:
Well, I finished logical exclusion updates, @leocwolter, I'm merging this PR!

Thanks for the reviews :-)